react-native-libraries.json

```diff
@@ -10356,5 +10356,15 @@
     "android": true,
     "web": true,
     "newArchitecture": true
+  },
+  {
+    "githubUrl": "https://github.com/kesha-antonov/react-native-background-downloader",
+    "npmPkg": "@kesha-antonov/react-native-background-downloader",
+    "examples": [
+      "https://github.com/kesha-antonov/react-native-background-downloader/tree/master/example"
+    ],
+    "ios": true,
+    "android": true,
+    "macos": true
```

Comment:
Hello @kesha-antonov, thank you for adding the new library! 👍

Entry looks good, however, we require an example when declaring out-of-tree platform support, like macOS, and example you have added do not include mac app. More information can be found in the Readme file section:
* https://github.com/react-native-community/directory?tab=readme-ov-file#%EF%B8%8F-out-of-tree-platforms

There are two ways to unblock this PR, either you can add macOS to the repository example, or the platform can be removed, and re-added in the future, when the example is updated or new one added.

Reply:
Removed macos